### PR TITLE
contributing guide

### PR DESCRIPTION
**Pull request by runzexia**
Signed-off-by: runzexia <runzexia@yunify.com>

**What type of PR is this?**
/kind docs
/kind cleanup

Clean up the previous contributor documentation.
Docs about build kubesphere core locally and how to submit a pull request.
/cc @zryfish @FeynmanZhou

### CODE REVIEW

File: docs/en/guides/How-to-build.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+> Note: KubeSphere uses `go module` to manage dependencies, but the development process still relies on `GOPATH`
+> In the CRD development process, you need to use tools to automatically generate code. The tools used by KubeSphere still need to rely on `GOPATH`. 
+> For Chinese contributors who are going to pull the go module, we recommend you to use [goproxy.cn](https://goproxy.cn) as the proxy.
+
+## Building KubeSphere Core on a local OS/shell environment
+
+### For Quick Taste Binary
+
+When you go get KubeSphere, you can choose the version you want to get: `go get kubesphere.io/kubesphere@version-you-want`
```
[the remark this answers is not in the document]
Comment (runzexia) [reply docs/en/guides/How-to-build.md]: https://github.com/kubesphere/kubesphere/pull/1551/files#diff-67a284ea5e9992bf6c176e0678c768dbR51
The following example shows what version means

File: docs/en/guides/How-to-build.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+
+> Note: Before getting KubeSphere, you need to synchronize the contents of the `replace` section of the go.mod file of the KubeSphere you want to version.
```
[the remark this answers is not in the document]
Comment (runzexia) [reply docs/en/guides/How-to-build.md]: Here we use the go get command to download kubesphere, so here I use `getting`.
The relevant replacement part refers to the content of the following file.
https://github.com/kubesphere/kubesphere/blob/master/go.mod#L155

File: docs/en/guides/How-to-run-KubeSphere-core.md [hunk 3]
```diff
@@ -0,0 +1,116 @@
+# How to run KubeSphere core in local
+
+This document will explain how to run KubeSphere apiserver locally.
+
+> Modules similar to KubeSphere apiserver are KubeSphere controller-manageer, Kubesphere iam (Aka KubeSphere account), KubeSphere api-gateway.
```
[the remark this answers is not in the document]
Comment (FeynmanZhou) [reply docs/en/guides/How-to-run-KubeSphere-core.md]: @runzexia Others may be confused for “Aka”, is it possible replace it with a clear vocabulary?

File: docs/en/guides/How-to-build.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (FeynmanZhou) [reply docs/en/guides/How-to-build.md]: The version in `GO111MODULE=on go get kubesphere.io/kubesphere@d649e3d0bbc64bfba18816c904819e4850d021e0`  looks like a series of random characters, still not clear about how to choose it as I want.

**Comment by runzexia** [comment]
@zryfish could we land these ？

**Pull request merged**